src/shared/components/structureViewer/StructureViewerPanel.tsx

```diff
@@ -508,6 +530,24 @@ export default class StructureViewerPanel extends React.Component<IStructureView
         }
     }
 
+    @computed get structureViewerBounds(): {width: number|string, height: number|string} {
+        let width: number|string;
+        let height: number|string;
+
+        // if 3Dmol container div is not initialized yet, just set to a default value: width=auto; height=350
+        // otherwise toggle the size
+        if (this.isIncreasedSize) {
```

Comment:
its definitely because `_3dMolDiv` is not observable :) if written like that, on the first passthrough it only encounters `this._3dMolDiv`, `width`, and `height`, thus it never registers as having to do a recomputation, even when `_3dMolDiv` becomes defined.